agent/consul/state/catalog_oss.go

```diff
@@ -208,14 +208,18 @@ func (s *Store) catalogInsertService(tx *memdb.Txn, svc *structs.ServiceNode) er
 		return fmt.Errorf("failed inserting service: %s", err)
 	}
 
-	// overall services index
-	if err := tx.Insert("index", &IndexEntry{"services", svc.ModifyIndex}); err != nil {
```

Comment:
Was this an older, manual transaction style before the helper method was added?

Reply:
Yep